app/scripts/modules/kubernetes/src/v2/manifest/selector/selector.component.ts

```diff
@@ -0,0 +1,66 @@
+import { IComponentOptions, IQService, IController, module } from 'angular';
+
+import { AccountService, IAccountDetails } from '@spinnaker/core';
+import { KUBERNETES_MANIFEST_LABEL_EDITOR } from './labelEditor/labelEditor.component';
+import { IManifestSelector } from './IManifestSelector';
+
+class KubernetesManifestSelectorCtrl implements IController {
+  public selector: IManifestSelector;
+  public accounts: IAccountDetails;
+  public kindsMetadata: string;
+
+  constructor(private $q: IQService, private accountService: AccountService) {
+    'ngInject';
+    const dataToFetch = {
+      accounts: this.accountService.getAllAccountDetailsForProvider('kubernetes', 'v2'),
+    };
+    this.$q.all(dataToFetch)
+      .then((fetchedData: any) => {
+        this.accounts = fetchedData.accounts;
+    });
```

Comment:
Please indent

Reply:
These stages usually pick an account for you (first try to use the default account in `settings.js` if it's included in the list of accounts, then just the first account in the list.